shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/consumercart/ConsumerCart.kt

```diff
@@ -3,61 +3,40 @@ package com.bunbeauty.shared.presentation.consumercart
 import com.bunbeauty.shared.presentation.base.BaseAction
 import com.bunbeauty.shared.presentation.base.BaseEvent
 import com.bunbeauty.shared.presentation.base.BaseViewDataState
-import com.bunbeauty.shared.presentation.menu.MenuProductItem
+import com.bunbeauty.shared.presentation.menu.model.MenuItem
+import com.bunbeauty.shared.presentation.motivation.MotivationData
 import com.bunbeauty.shared.presentation.product_details.ProductDetailsOpenedFrom
 
 interface ConsumerCart {
 
-    data class ViewDataState(
-        val consumerCartData: ConsumerCartData?,
-        val screenState: ScreenState,
+    data class DataState(
+        val state: State,
+        val motivation: MotivationData?,
+        val cartProductItemList: List<CartProductItem>,
+        val recommendationList: List<MenuItem.Product>,
+        val discount: String?,
+        val oldTotalCost: String?,
+        val newTotalCost: String,
     ) : BaseViewDataState {
```

Comment:
BaseDataState insted of BaseViewDataState

Reply:
✅ fixed